prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/channel/ChannelAddCommand.java

```diff
@@ -43,36 +45,55 @@ public class ChannelAddCommand extends AbstractCommand {
     @CommandLine.Option(names = CliConstants.CHANNEL_NAME, required = true)
     private String channelName;
 
-    @CommandLine.Option(names = CliConstants.CHANNEL_MANIFEST, required = true)
-    private String gavUrlOrPath;
+    @CommandLine.ArgGroup(exclusive = false)
+    ChannelGroup channelGroup;
+
+    static class ChannelGroup {
```

Comment:
Can you try to make them exclusive with channelLocation? I think it should be possible using a nested ArgGroup.